server/src/test/java/com/exasol/testsetupabstraction/server/TestSetupServerTest.java

```diff
@@ -0,0 +1,65 @@
+package com.exasol.testsetupabstraction.server;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.*;
+
+import com.exasol.exasoltestsetup.*;
+
+class TestSetupServerTest {
```

Comment:
If coverage is not good enough, add tests with invalid request data, e.g. missing params.